src/ansi-c/literals/unescape_string.cpp

```diff
@@ -44,15 +44,15 @@ std::basic_string<T> unescape_string_templ(const std::string &src)
 
   for(unsigned i=0; i<src.size(); i++)
   {
-    T ch=(unsigned char)src[i];
+    T ch = narrow_cast<T>(narrow_cast<unsigned char>(src[i]));
```

Comment:
:question: Why narrow to unsigned char first? :mouse:

Reply:
This is to preserve the existing behaviour, as `T` might actually be wider than `unsigned char`. That said, I guess `narrow_cast` would fail in a case where that makes a difference, so I can probably get rid of it.

Reply:
Nested cast removed.